staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/BUILD

```diff
@@ -1,9 +1,10 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
 
 go_library(
     name = "go_default_library",
     srcs = [
         "converter.go",
+        "monitoring.go",
```

Comment:
monitoring or metrics?

Reply:
Monitoring is a class that adds metrics, I have no strong feeling about either but prefer monitoring a little.